circle.yml

```diff
@@ -1,28 +1,63 @@
 machine:
   environment:
     # We need to set this variable to let python see the package installed through apt-get
-    PYTHONPATH: "/usr/lib/python2.7/dist-packages"
-dependencies:
-  cache_directories:
+    # PYTHONPATH: "/usr//usr/lib/python2.7/dist-packages"
+    PYTHON_VERSION: "2.6"
+    NUMPY_VERSION: "1.6.2"
+    SCIPY_VERSION: "0.12.0"
+    CYTHON_VERSION: "0.21"
```

Comment:
Why building the doc with such old version of Python, numpy and scipy?

Maybe just use the latest by not including any version number for those libraries in the conda create command.